File: OMGStarWars/app/src/main/java/dev/radley/omgstarwars/ApplicationController.java
package dev.radley.omgstarwars;

import android.app.Application;

import timber.log.Timber;

public class ApplicationController extends Application {

    /**
     * Add Timber support for logging
     */
    @Override
    public void onCreate() {
        super.onCreate();
        if(BuildConfig.DEBUG){
            Timber.plant(new Timber.DebugTree());
        }
    }
}

File: OMGStarWars/app/src/main/java/dev/radley/omgstarwars/ApplicationController.kt
package dev.radley.omgstarwars

import android.app.Application

import timber.log.Timber

class ApplicationController : Application() {

    /**
     * Add Timber support for logging
     */
    override fun onCreate() {
        super.onCreate()
        if (BuildConfig.DEBUG) {
            Timber.plant(Timber.DebugTree())
        }
    }
}
